src/profile-logic/profile-data.js

```diff
@@ -2695,3 +2711,33 @@ export function hasThreadKeys(
   }
   return true;
 }
+
+/**
+ * TODO: write
+ */
+export function computeMaxThreadCPU(
```

Comment:
On platforms where the cpuDelta is given as a time, we should always use "1" as the maxThreadCpu. Or, ideally, not even enter a code path where we divide by maxThreadCpu, to make it clear that there is no dynamic scaling.
This is why I [proposed having separate field names](https://phabricator.services.mozilla.com/D99413?id=377596#inline-560874) depending on whether the value is a time or not.